exasol_script_languages_container_ci/lib/release.py

```diff
@@ -41,12 +41,12 @@ def release(ctx: click.Context,
 
     ci_build(ctx, flavor_path=flavor_path, rebuild=True, build_docker_repository="",
              commit_sha="", docker_user="", docker_password="")
-    ci_test(ctx, check_for_skip_test=False, flavor_path=flavor_path, git_access=git_access)
+    execute_test(ctx, flavor_path=flavor_path)
```

Comment:
Why did you switch to execute_test here instead using ci_tests

Reply:
execute_test is used by ci_tests and the release tests

Reply:
I guess, you wanted to make sure, that the skip test logic in ci_tests doesn't affect the release

Reply:
Yes, exactly.